### DISCUSSION TITLE
soup 2.3.7 => 2.3.100 error

### DISCUSSION THREAD

**Discussion by elenezet** [Security-Onion-Solutions/securityonion#7065] [General] [answered]
I try to upgrade SO 2.3.7 to 2.3.100 but i've an error :
curl: (6) Could not resolve host: raw.githubusercontent.com; Unknown error
Command failed with error 6
Could not pull signature key file, please ensure connectivity to https://raw.githubusercontent.com

Do you have a idear?

**Comment by dougburks** [comment, marked as answer]
Verify that your machine has Internet access as described here:
https://docs.securityonion.net/en/2.3/firewall.html#internet-communication

**Comment by elenezet** [comment]
Ok i'll add all this sites on whitelistbut i'he alway the same error : 
[suricatux@suricatux-1-a sbin]$ curl -I https://raw.githubusercontent.com
HTTP/1.1 200 Connection established

HTTP/1.1 301 Moved Permanently
Connection: keep-alive
Content-Length: 0
Content-Security-Policy: default-src 'none'; style-src 'unsafe-inline'; sandbox
Location: https://github.com/
Strict-Transport-Security: max-age=31536000
X-Content-Type-Options: nosniff
X-Frame-Options: deny
X-XSS-Protection: 1; mode=block
X-GitHub-Request-Id: 823C:CCEF:6404AB:67B92E:62023888
Accept-Ranges: bytes
Date: Tue, 08 Feb 2022 09:37:21 GMT
Via: 1.1 varnish
X-Served-By: cache-cdg20723-CDG
X-Cache: HIT
X-Cache-Hits: 3
X-Timer: S1644313042.605462,VS0,VE0
Vary: Authorization,Accept-Encoding,Origin
Access-Control-Allow-Origin: *
X-Fastly-Request-ID: 29c01ebec678f3eddcd16bca4dde13451148a66d
Expires: Tue, 08 Feb 2022 09:42:21 GMT
Source-Age: 329

Summary for local
------------
Succeeded: 6 (changed=1)
Failed:    0
------------
Total states run:     6
Total run time:   2.162 s
curl: (6) Could not resolve host: raw.githubusercontent.com; Unknown error
Command failed with error 6
Could not pull signature key file, please ensure connectivity to https://raw.githubusercontent.com

If anyone have an idea ?

**Reply by dougburks** [reply]
> Could not resolve host: raw.githubusercontent.com; Unknown error

If it is saying `Could not resolve host`, that seems like a DNS issue on your network.

**Comment by elenezet** [comment]
yep it look like curl can't resolv raw.githubusercontent.com. I think there is a proxy error.

**Comment by elenezet** [comment]
Up ... with export proxy_http it work's
